imagersite/imager_images/templates/imager_images/album_edit.html

```diff
@@ -0,0 +1,44 @@
+{% extends "base.html" %}
+{% load images_filters %}
```

Comment:
Jesse gets the credit for pointing me in this direction. Some cool potential there!